pretty.js

```diff
@@ -46,19 +53,40 @@ function _lpadzero (aTarget, aLength, aPadChar) {
 function withSpaces (value, eol) {
   var lines = value.split(/\r?\n/)
   for (var i = 1; i < lines.length; i++) {
-    lines[i] = '    ' + lines[i]
+    lines[i] = indent(4) + lines[i]
   }
   return lines.join(eol)
 }
 
-function filter (value, messageKey, eol) {
+function indent (n) {
+  return Array(n + 1).join(' ')
+}
+
+function filter (value, messageKey, eol, errorLikeObjectKeys) {
```

Comment:
I had to add an argument to `filter`.

To not break its signature, I appended it to the arg list, but I think the order now is kind of awkward and the function is not tetradic :/. Maybe we should consider refactor this.

Reply:
I'm good with tetradic.